config/arbitrum-sepolia-staging.json

```diff
@@ -0,0 +1,279 @@
+      "blockNumber": 937420
+    },
+    {
+      "address": "0x3b1Fe77D72e2DE15EF3A1ff83176e9F9af9E292A",
+      "name": "Arbitrum sepolia staging shared minter: MinterDAExpHolderV5",
+      "blockNumber": 937420
+    },
+    {
+      "address": "0x6feb0aB02A85d2C3c5a79B822f0a00061cBBb040",
+      "name": "Arbitrum sepolia staging shared minter: MinterDALinHolderV5",
+      "blockNumber": 937420
+    }
+  ],
+  "merkleLibContracts": [
+    {
+      "address": "0x2146A41F2c1432895d6D0ADF9c60BF0a226Bac0e",
+      "name": "Arbitrum sepolia staging shared minter: MinterSetPriceMerkleV5",
+      "blockNumber": 937420
+    }
+  ],
+  "DALibContracts": [
+    {
+      "address": "0xA6AE3B1c0177C9addA7E9f57Bb96686F892aFe10",
+      "name": "Arbitrum sepolia staging shared minter: MinterDAExpV5",
+      "blockNumber": 937420
+    },
+    {
+      "address": "0x06Bdff0A69d1b8E18C5B27b03945f024d3490691",
+      "name": "Arbitrum sepolia staging shared minter: MinterDALinV5",
+      "blockNumber": 937420
+    },
+    {
+      "address": "0xa319C382a702682129fcbF55d514E61a16f97f9c",
+      "name": "Arbitrum sepolia staging shared minter: MinterDAExpSettlementV3",
+      "blockNumber": 937420
+    },
+    {
+      "address": "0x3b1Fe77D72e2DE15EF3A1ff83176e9F9af9E292A",
+      "name": "Arbitrum sepolia staging shared minter: MinterDAExpHolderV5",
+      "blockNumber": 937420
+    },
+    {
+      "address": "0x6feb0aB02A85d2C3c5a79B822f0a00061cBBb040",
+      "name": "Arbitrum sepolia staging shared minter: MinterDALinHolderV5",
+      "blockNumber": 937420
+    }
+  ],
+  "DAExpLibContracts": [
+    {
+      "address": "0xA6AE3B1c0177C9addA7E9f57Bb96686F892aFe10",
+      "name": "Arbitrum sepolia staging shared minter: MinterDAExpV5",
+      "blockNumber": 937420
+    },
+    {
+      "address": "0xa319C382a702682129fcbF55d514E61a16f97f9c",
+      "name": "Arbitrum sepolia staging shared minter: MinterDAExpSettlementV3",
+      "blockNumber": 937420
+    },
+    {
+      "address": "0x3b1Fe77D72e2DE15EF3A1ff83176e9F9af9E292A",
+      "name": "Arbitrum sepolia staging shared minter: MinterDAExpHolderV5",
+      "blockNumber": 937420
+    }
+  ],
+  "DALinLibContracts": [
+    {
+      "address": "0x06Bdff0A69d1b8E18C5B27b03945f024d3490691",
+      "name": "Arbitrum sepolia staging shared minter: MinterDALinV5",
+      "blockNumber": 937420
+    },
+    {
+      "address": "0x6feb0aB02A85d2C3c5a79B822f0a00061cBBb040",
+      "name": "Arbitrum sepolia staging shared minter: MinterDALinHolderV5",
+      "blockNumber": 937420
+    }
+  ],
+  "settlementExpLibContracts": [
+    {
+      "address": "0xa319C382a702682129fcbF55d514E61a16f97f9c",
+      "name": "Arbitrum sepolia staging shared minter: MinterDAExpSettlementV3",
+      "blockNumber": 937420
+    }
+  ],
+  "adminACLV0Contracts": [
+    {
+      "address": "0x3b30d421a6dA95694EaaE09971424F15Eb375269",
+      "name": "Admin ACL for shared minter filter",
+      "blockNumber": 936767
+    }
+  ],
```

Comment:
We shouldn't need to track adminACL for the shared minter filter - only should index the ones associated with core contracts.